### PR TITLE
[ENHANCEMENT] Reduce navbars size

### PR DESCRIPTION
**Pull request by AntoineThebaud**
# Description

Reduce the size of the various navbars (top one, the one with breadcrumb & timerange controls in read mode, the one with variables dropdowns & timerange controls in edit mode, also the sticky version of the latter) to spare vertical space for the dashboard content.

# Screenshots

before/after comparison:

image

image

2023-10-20_07h42_22

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [x] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [x] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [ ] Visual tests are stable and unlikely to be flaky. See [Storybook](https://github.com/perses/perses/tree/main/ui/storybook#visual-tests) and [e2e](https://github.com/perses/perses/tree/main/ui/e2e#visual-tests) docs for more details. Common issues include:
  - Is the data inconsistent? You need to mock API requests.
  - Does the time change? You need to use consistent time values or mock time utilities.
  - Does it have loading states? You need to wait for loading to complete.

### CODE REVIEW

File: ui/app/src/components/Header.tsx [hunk 1]
```diff
@@ -147,6 +147,11 @@ export default function Header(): JSX.Element {
       <Toolbar
         sx={{
           backgroundColor: (theme) => theme.palette.designSystem.blue[700],
+          '&': {
+            minHeight: '40px',
+            paddingLeft: 0,
+            paddingRight: 0.75,
```
Comment (Gladorme) [inline ui/app/src/components/Header.tsx:153]:
```suggestion
            paddingRight: 2,
```
In order to have the same padding as "body" content

**Comment by Gladorme** [comment]
I think, we could also remove the paddingTop of `perses/ui/dashboards/src/components/DashboardToolbar/DashboardToolbar:tsx`at ligne 119

File: ui/app/src/components/Header.tsx (hunk 1 above)
Comment (AntoineThebaud) [reply ui/app/src/components/Header.tsx:153]: Yeah ok it's a matter of either having the button zone aligned with the rest (padding 2)
image
or with the slider itself (padding 0.75)
image

Personally I have a slight preference for the 2nd one even it looks a bit like a hack here I agree :D.

**Review by Nexucis — APPROVED** [review]
nice !

**Review by Gladorme — APPROVED** [review]

**Comment by Nexucis** [comment]
by the way does it reduce the navbar size of the variable ?

File: ui/app/src/components/Header.tsx (hunk 1 above)
Comment (Gladorme) [reply ui/app/src/components/Header.tsx:153]: This hover effect is pretty annoying, ok for me

**Comment by AntoineThebaud** [comment]
> by the way does it reduce the navbar size of the variable ?

You mean the sticky bar I assume? Then yes, actually my bad I forgot to put a screenshot for this one, you can see one now in the PR description.

**Comment by nicolastakashi** [comment]
Much much better 🥳

**Review by nicolastakashi — APPROVED** [review]

**Pull request merged**